### ISSUE TITLE
Toc not visible with 'Default 2022 presentation style

### ISSUE THREAD

**Issue by davidtr38** [EWSoftware/SHFB#1018]
Hello, Toc not visible with 'Default 2022 presentation style.
I use:
- lastest version
- Plug-ins : content only
- Build > Presentation : Default 2022
- Help file format : Website

It's ok if i select 'VS2013' in presnetation style.
My project is very simple for the moment juste 3 pages (12 main level and one child)

Is there a way to show the TOC for the 'Default 2022' style ?

**Comment by EWSoftware** [comment]
If you're trying to view it via the file system, the browser will block the scripts.  Run it using the View Website option so that it starts a local web server to view it.

**Comment by davidtr38** [comment]
It doesn't work. I also tested under Apache and it's the same result.
(no error on build output)

**Comment by EWSoftware** [comment]
Sorry, I don't know what to tell you.  It does work as you can see from the [SHFB help](https://ewsoftware.github.io/SHFB/html/bd1ddb51-1c4f-434f-bb1a-ce2135d3a909.htm).  Can you post a screenshot to show what you are seeing?

**Comment by davidtr38** [comment]
Thank you for the time spent.
here screenshots for default2022 and vs2013 style (with build logs)
thanks

**Screenshot wifth Default2022 Style**
![HC_With_Default2022](https://github.com/EWSoftware/SHFB/assets/144314836/4c3b3673-c917-4279-851a-1ffc42935c14)

**Screenshot with VS2013 Style**
![HC_With_VS2013](https://github.com/EWSoftware/SHFB/assets/144314836/bb99b9fd-ef95-44a6-bee9-88bc23b1df1c)

**Build Log Default2022**
[Log_With_Default2022.txt](https://github.com/EWSoftware/SHFB/files/12555852/Log_With_Default2022.txt)

**Build Log VS2013**
[Log_With_VS2013.txt](https://github.com/EWSoftware/SHFB/files/12555855/Log_With_VS2013.txt)

**Comment by davidtr38** [comment]
Here the project. Thanks
[Sandcastle.zip](https://github.com/EWSoftware/SHFB/files/12568272/Sandcastle.zip)

**Comment by YImhof** [comment]
Hello,

I encountered the same problem, using SHFB 2023.7.8.0 as VS2022 (17.7.4) integration package.

I tried to change the configuration in several ways, inspiring me from source project file [shfbproj](https://github.com/EWSoftware/SHFB/blob/master/Documentation/SandcastleBuilder/SandcastleBuilder.shfbproj), but I didn't find how to solve this. Both in the header and in the TOC column, HTML tags `nav` and `ul` are present, but there is no item list (`il` tag).
